### DISCUSSION TITLE
Organizations UpdateMany

### DISCUSSION THREAD

**Discussion by ZamTheman** [mozts2005/ZendeskApi_v2#491] [Features]
Any plans to add a UpdateMany for organizations?

Thanks for great work!

**Comment by mozts2005** [comment]
at this time no. but if you would like to add it we would be happy to take a PR for it

**Reply by ZamTheman** [reply]
I cloned the repo but the Unit Tests are failing due to authentication errors. Have I missed something?

After testing using Postman it seems like token might be expired?
{
    "error": "invalid_token",
    "error_description": "The access token provided is expired, revoked, malformed or invalid for other reasons."
}

**Reply by mozts2005** [reply]
the token has been updated please update and try again.
